active/teachingtip/TeachingTipSpec.md

```diff
@@ -0,0 +1,565 @@
+| Motion | * Tips have built in open and close animations that can be customizable using Storyboards.|
+| Pointer/Hero Content Avoidance | * To avoid the visual oddity of the Pointer emerging from Hero Content, the Pointer and Hero Content will attempt to avoid intersecting using the following rules: <br> &nbsp;&nbsp;&nbsp;&nbsp; * Move the hero content to Top or Bottom (Disabled when HeroContentPlacement is not auto). <br> &nbsp;&nbsp;&nbsp;&nbsp; * Shift the beak along the edge of the tip (Disabled when the placement of the tip is edge aligned). <br> &nbsp;&nbsp;&nbsp;&nbsp; * Change the placement of the tip (Disabled when the tip placement is not auto).  |
+| Out of Window Bounds | * Tips can escape window bounds on newer OS versions via the ShouldConstrainToRootBounds property.  When this property is set to false, the tip uses screen boundaries instead of window boundaries during its placement algorithm. |
+
+### Gamepad Interaction 
+| State | Action |
+|:---|:---|
+| Tip first appears and is not in focus | No action is needed invoke the tip. |
+| Tip is first focused on | Tip steals focus and returns focus to the previous element upon closing. |
+| Tip is navigated | Directional navigation: <br> Will spatially navigate focus across actionable items (without respect to group).  <br> <br> A Button: <br> Will interact with the item in focus, such as "press" the action or close button. <br><br> B Button: <br> Will result in closing the tip. |
+| Tip is dismissed | 1. Header "X" Close Button is pressed. <br> 2. Footer Close Button is pressed. <br> 3. Action Button is pressed. <br> 4. B Button returns focus to the element previously in focus. |
+
+### Accessibility
+
+#### UI Automation Patterns 
+
+TeachingTip will alternate between Pane for persistent TeachingTips and Window for light-dismiss enabled TeachingTips with IScrollProvider for the (conditionally) scrollable content area within TeachingTip. 
+
+#### Keyboarding and Narrator Experience 
+| State | Action | Narrator |
+|:---|:---|:---|
+| Tip first appears and is not in focus | No action is needed invoke the tip. |  Persistent tip: <br> “New tip. It says..." + Title + Subtitle <br><br> Light-dismiss tip: <br> “New light-dismiss tip. It says..." Title + Subtitle | 
+| Tip is first focused on | Tip steals focus and returns focus to the previous element upon closing. | Title + Button in Focus |
+| Tip is tabbed through | Tab Button: <br> Will go through all actionable items, regardless of group, in order. When tab is pressed on the last element in the teaching tip, focus will cycle to the first element in the teaching tip.  <br> <br> Left + Right Arrow Keys: <br> Can be used to navigate between the footer Action and Close buttons if both are present. <br><br> Escape: <br> Will result in closing the tip. | | 
+| Tip is dismissed | 1. X Button is pressed. <br> 2. Close Button is pressed. <br> 3. Action Button is pressed. <br> Tab focus returns to the predecessor. | | 
```

Comment:
Do we need to inform Narrator what object the tip is pointing to?

Reply:
Added requirement to inform Narrator what object the tip is pointing to [here](https://github.com/Microsoft/microsoft-ui-xaml-specs/commit/71881ccf08475b55e37409ec3f9e590e2e8c441d).